CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+1.4.0 / 2019-08-28
+==================
+
+Bug fixes:
+* Don't use TABs, when triggering `change` JS event upon input value is change (fixes auto-complete control testing in Google Chrome)
+* Fixed inability to manipulate windows when Selenium 3 with Firefox GeckoDriver was used
+* The `clickOnElement` method wasn't working when Selenium 3 with Firefox GeckoDriver was used
+
+Testsuite:
```

Comment:
Please move this section at the bottom (as in other versions).

Reply:
moved testsuite before Misc